### ISSUE TITLE
SD: add functionality to Outfalls for Tidal Curve and Time Series

### ISSUE THREAD

**Issue by FLO-2DJJ** [FLO-2DSoftware/qgis-flo-2d-plugin#854]
See how EPA SWMM does it. And also in Inlets dialog (under External Inflow).

**Comment by FLO-2DJJ** [comment]
For Tidal Curves and Time Series consider the options of create the data and/or read it for external files.

**Comment by FLO-2DJJ** [comment]
This is the same issue #883

**Comment by FLO-2DJJ** [comment]
This enhancement has been extensively programmed in the last 3 weeks.

**Comment by FLO-2DJJ** [comment]
@FLO-2DNoemi, @FLO-2DKaren please test this issue. 

- Requires a new fresh project since the database has been extended with 2 new tables: 'swmm_tidal_curve' and  'swmm_tidal_curve_data' for a new tidal curve  boundary condition for outfalls.
- Tables 'swmm_inflow_time_series' and 'swmm_inflow_time_series_data' were renamed to 'swmm_time_series'  and 'swmm_time_series_data' respectively to account to the fact that now time series are assigned to both inflows and outfalls.
- Import .INP and export .INP code has also been extended to include the above changes.
- Tests compatibility with EPA SWMM 5.0 are recomended.

**Comment by FLO-2DKaren** [comment]
@FLO-2DNoemi @FLO-2DJJ 
JJ, did you also consider the import *.inp button?

**Comment by FLO-2DJJ** [comment]
> 

Yes, the import .INP button also considers the required changes to the tidal curves and time series.

**Comment by FLO-2DJJ** [comment]
Fixed. @FLO-2DKaren, @FLO-2DNoemi please test it and report to close issue.
